core/libp2p/multi/CMakeLists.txt

```diff
@@ -3,4 +3,26 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+add_subdirectory(c-utils)
+
 add_subdirectory(multiaddress)
+
+add_library(uvarint
+    uvarint.cpp
+    uvarint.hpp
+    )
```

Comment:
Move uvarint this to common, since it is not specific for multi

Reply:
Actually, this version of varint is quite specific to multiformats and, as I understand, is gonna be used only for them, so that's why I decided to keep it in multi.

Reply:
Okay then